kebechet/managers/update/update.py

```diff
@@ -109,6 +109,7 @@ def __init__(self, *args, **kwargs):
         # We do API calls once for merge requests and we cache them for later use.
         self._cached_merge_requests = None
         self._pr_list = []
+        self.runtime_environment = "default"
```

Comment:
Why is this line added here?

Reply:
previously , self.runtime_environment was a string, being used in https://github.com/thoth-station/kebechet/blob/master/kebechet/managers/update/update.py#L761 and many more places.
Since we are converting it list of strings, do not want to break the code since it's already being used in various methods, created a variable to keep it running